tests/unit_tests/config/test_ert_config.py

```diff
@@ -707,8 +707,8 @@ def test_default_ens_path(tmpdir):
     "max_running_value, expected_error",
     [
         (100, None),  # positive integer
-        (-1, "QUEUE_OPTION MAX_RUNNING is negative"),  # negative integer
-        ("not_an_integer", "QUEUE_OPTION MAX_RUNNING is not an integer"),  # non-integer
+        (-1, "is not a valid positive integer"),  # negative integer
```

Comment:
Should we not assert that the string MAX_RUNNING is part of the error? (via regexp if needed)

Reply:
ah, tested further below. Do we need this test here then?

Reply:
Not sure, maybe we can skip it?

Reply:
The code below lacks testing of "-1". The happy path ("100") is probably tested within the hypothesis setup I suppose.